src/index.js

```diff
@@ -9,7 +9,9 @@ const { Primitives } = require('./primitives')
 const Vec3 = require('vec3').Vec3
 
 const io = require('socket.io-client')
-const socket = io()
+const socket = io('http://localhost/socket.io', {
```

Comment:
This doesn't seem correct, the host is not necessary localhost.

Reply:
in fact, if you don't change this line at all, you don't need the path and it just work.